codegen/protocol-test-codegen/build.gradle.kts

```diff
@@ -32,12 +32,12 @@ val enabledProtocols = listOf(
     ProtocolTest("aws-restjson", "aws.protocoltests.restjson#RestJson", "AWSRestJsonTestSDK"),
     ProtocolTest("rest-xml", "aws.protocoltests.restxml#RestXml", "RestXmlTestSDK"),
     ProtocolTest("rest-xml-xmlns", "aws.protocoltests.restxml.xmlns#RestXmlWithNamespace", "RestXmlWithNamespaceTestSDK"),
-    ProtocolTest("aws-query", "aws.protocoltests.query#AwsQuery", "AWSQueryTestSDK")
+    ProtocolTest("aws-query", "aws.protocoltests.query#AwsQuery", "AWSQueryTestSDK"),
 
     // service specific tests
     //ProtocolTest("apigateway", "com.amazonaws.apigateway#BackplaneControlService"),
     //ProtocolTest("glacier", "com.amazonaws.glacier#Glacier", "GlacierTestSDK")
-    //ProtocolTest("s3", "com.amazonaws.s3#AmazonS3", "S3TestSDK")
+    ProtocolTest("s3", "com.amazonaws.s3#AmazonS3", "S3TestSDK")
```

Comment:
how exciting!

Reply:
yes, exciting, but less exciting:
```
        val testsToIgnore = setOf(
            "S3DefaultAddressing",
            "S3VirtualHostAddressing",
            "S3PathAddressing",
            "S3VirtualHostDualstackAddressing",
            "S3VirtualHostAccelerateAddressing",
            "S3VirtualHostDualstackAccelerateAddressing",
            "S3OperationAddressingPreferred"
        )
```
:)

Reply:
ya I saw this next haha but still it means we are making progress and chipping away at s3 customizations so still exciting